build.ps1

```diff
@@ -1,8 +1,10 @@
 $ErrorActionPreference = "Stop"
 
-dotnet test SentryNoSamples.slnf -c Release -l "GitHubActions;report-warnings=false" `
+$testLogger = if ($env:GITHUB_ACTIONS -eq "true") {"GitHubActions;report-warnings=false"} else {"console"}
+
+dotnet test SentryWindows.slnf -c Release -l $testLogger `
     /p:CopyLocalLockFileAssemblies=true
 if ($LASTEXITCODE -ne 0) { exit 1 }
 
-dotnet pack SentryNoSamples.slnf -c Release --no-build /p:ContinuousIntegrationBuild=true
+dotnet pack SentryWindows.slnf -c Release --no-build
```

Comment:
We might want to move to WindowsNoSample here but for now just filtering by what runs on Windows

Reply:
Deleted the /p=

https://github.com/getsentry/sentry-dotnet/blob/b8e90c4f768b345a41c0fb07ec7fd65a3a023b67/Directory.Build.props#L20-L22